src/pages/loginView.js

```diff
@@ -1,7 +1,44 @@
-export default function LoginView( ) {
-    return (
-        <div>
-            <h2>Login</h2>
+import React, {useState} from 'react';
+import Button from '../components/global/Button';
+import TextInput from '../components/global/TextInput';
+import LoginImage from '../assets/images/LoginImage';
+import {Link} from 'react-router-dom';
+
+import '../stylesheets/loginView.scss';
+
+export default function LoginView() {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const login = function () {
+    console.log(email, password);
```

Comment:
For other reviewers, this a placeholder while we put together the proper authentication functionality